drivers/performance_metrics/app/models/performance_metrics/report.rb

```diff
@@ -679,21 +694,27 @@ def report_labels
 
     private def run_rrh
       rrh_filter = WarehouseReport::Outcomes::OutcomesFilter.new(user_id: filter.user_id)
-      rrh_filter.update(filter.to_h)
+      options = filter.to_h
+      options.delete(:comparison_pattern)
+      rrh_filter.update(options)
```

Comment:
It looks like this pattern gets repeated 4 times. Might be good to extract it to its own method.